src/__tests__/__views__/ReportArticle.test.js

```diff
@@ -0,0 +1,86 @@
+import React from "react";
+
+import { shallow } from "enzyme";
+import Input from "../../components/common/Inputs/TextInput";
+import Button from "../../components/common/Buttons/BasicButton";
+import { ReportArticle, mapStateToProps } from "../../views/ReportArticle";
+
+const shallowSetup = () => {
+  const props = {
+    response: "hello world",
+    match: {
+      params: {
+        slug: "hello-world"
+      }
+    },
+    reportedArticle: jest.fn()
+  };
+  const wrapper = shallow(<ReportArticle {...props} />);
+
+  return {
+    wrapper,
+    props
+  };
+};
+
+describe("Report article component", () => {
+  test("should render the component for creating a report", () => {
+    const { wrapper } = shallowSetup();
```

Comment:
This line is repeated in many places! Can you declare it as a global variable or in a before...